pkgs/development/python-modules/imgsize/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "imgsize";
+  version = "2.1";
+
+  format = "wheel";
```

Comment:
Please avoid wheels, even when pure ones. Its preferable to fetch from GitHub in that case.

Reply:
Done.  Is there a general policy against wheels?  What's wrong with them?

Reply:
usually testing, as wheel will usually filter out tests, and they are nice to ensure that the package is still valid with the dependencies we give it (which is just the interpreter in this case)

Other reason is that the wheel may contained compiled code, and it's generally prefered to use the toolchain that nixpkgs provides

Reply:
Thanks for the explanation!